### PR TITLE
Add single-directional touch action values

### PR DESCRIPTION
**Pull request by NavidZ**
***
<a href="https://pr-preview.s3.amazonaws.com/NavidZ/pointerevents/pull/294.html" title="Last updated on Aug 29, 2019, 3:46 PM UTC (9204108)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/294/eff9f15...NavidZ:9204108.html" title="Last updated on Aug 29, 2019, 3:46 PM UTC (9204108)">Diff</a>

### CODE REVIEW

**Comment by NavidZ** [comment]
@smaug---- @BoCupp-Microsoft @graouts @patrickhlauke any feedback is appreciated.
This feature was already implemented/shipped in Chrome like 2 years ago and you can give it a try there if you like.

@plehegar, @patrickhlauke I changed the PE level to version 3 in this pull request and added links and whatnot to PEv2. Let me know if we are planning it some other way or whether it is alright from W3C point of view to have it like this. I assumed the published version of V2 is already hosted as an html file and has nothing to do with the top commit of this repo. So we can just change the code here and move to the V3. But feel free to correct me if I'm wrong. I can change this PL to whatever you suggest.

**Comment by patrickhlauke** [comment]
changed this to use pe3 as base branch - just so we can work on integrating the extension stuff and touch action values etc into it and then plop the v3 rough version into gh-pages in one fell swoop (not that it makes much of a difference i guess)

**Comment by patrickhlauke** [comment]
discussed in PEWG call today, no point working separately in pe3 branch. we'll just do it straight in gh-pages...

**Comment by NavidZ** [comment]
@smaug---- @BoCupp-Microsoft @graouts any objections/comments regarding the new values? Can we merge this PR as is?

File: index.html [hunk 1]
```diff
@@ -56,7 +56,8 @@
     </section>
 
     <section id="sotd">
-        <p>This specification is an update to [[PointerEvents]] which was shipped broadly only by Microsoft Internet Explorer and Microsoft Edge (though a further independent and mostly interoperable implementation was present in a pre-release build of Mozilla Firefox when the Pointer Events specification was published as a W3C Recommendation). Level 2 includes editorial clarifications, new features and minor breaking changes that address certain limitations and concerns that have been raised about aspects of the design, in an effort to enable wider browser adoption.</p>
+        <p>This specification is an update to [[PointerEvents2]] which was shipped broadly by Google Chrome and Microsoft Edge and Mozzila Firefox.
+           Level 3 includes editorial clarifications, new features to enable more use cases, in an effort to enable wider developer and browser adoption.</p>
```
Comment (graouts) [inline index.html]: This change does not seem relevant to the new single-directional touch-action values. I think they should be moved to a different PR.

**Review by graouts — CHANGES_REQUESTED** [review]

**Review by patrickhlauke — APPROVED** [review]

**Comment by NavidZ** [comment]
@graouts this seems to be merged now. But feel free to still review the change and we can still incorporate your feedback if you don't agree with the current proposal.

**Comment by patrickhlauke** [comment]
oh sorry, i just assumed that since @graouts' comment was only related to the out-of-scope changes in this PR, that he was ok with the rest. if that wasn't the case, apologies Antoine

**Comment by graouts** [comment]
No worries, you assumed correctly.

**Comment by mustaqahmed** [comment]
We already have the following WPTs covering this PR: `pointerevent_touch-action-pan-{left,right,up,down}-css_touch.html`.

**Pull request merged**